src/Nri/Css/Extra.elm

```diff
@@ -14,3 +37,115 @@ import Css exposing (..)
 lineHeightNum : Float -> Mixin
 lineHeightNum height =
     property "line-height" <| toString height
+
+
+{-| Helper for content CSS attribute
+-}
+content : String -> Mixin
+content value =
+    property "content" <| "'" ++ value ++ "'"
+
+
+{-| z-index helper
+-}
+zIndex : Int -> Mixin
+zIndex value =
+    property "z-index" <| toString value
```

Comment:
I was doing that, but features/bugs are released in batches (2 weeks to a month at a time). I just wanted to have a place to throw these until the PRs get merged.